src/wavm/dynlink.cpp

```diff
@@ -73,7 +73,7 @@ WAVM_DEFINE_INTRINSIC_FUNCTION(env, "dlerror", I32, dlerror)
     faabric::util::getLogger()->debug("S - dlerror");
     std::string errorMessage("Wasm dynamic linking error. See logs");
 
-    wasm::WAVMWasmModule* module = getExecutingWAVMModule();
+    WAVMWasmModule* module = getExecutingWAVMModule();
```

Comment:
There seemed to be some unnecessary namespacing that started erroring when I changed the layout, so I removed it